app/javascript/article-form/elements/__tests__/__snapshots__/tags.test.jsx.snap

```diff
@@ -60,32 +60,10 @@ Object {
   "prevLen": 0,
   "searchResults": Array [
     Object {
-      "_highlightResult": Object {
```

Comment:
Stating the obvious probably, but this property on the object that is removed from the snapshot is Algolia specific metadata I imagine?

Reply:
Correct